docs/deployments/oc10-app.md

```diff
@@ -74,6 +79,10 @@ There are a few config values which need to be set in order for ownCloud Web to
 }
 ```
 
+{{< hint info >}}
+Please note that the `server` url needs to have a trailing slash.
+{{< /hint >}}
+
```

Comment:
not needed anymore, thanks to `\rtrim`

Reply:
That only applies to the `web.baseUrl`. config.json still needs the trailing slash for the `server` url. On the other hand, it makes more sense to enforce it when loading the server url. Will make a separate PR.